spec/lib/theme_settings_object_validator_spec.rb

```diff
@@ -112,6 +112,29 @@
           described_class.new(schema: schema, object: { float_property: "string" }).validate,
         ).to eq(float_property: ["must be a float"])
       end
+
+      it "should return the right hash of error messages when integer property does not satisfy min or max validations" do
```

Comment:
I think we should add tests for the happy path, i.e. when value does pass the validations.